src/components/product-card/ProductCard.jsx

```diff
@@ -0,0 +1,38 @@
+import { CartIcon, HeartIcon } from "../../assets/icons";
+import { sold_out_img } from "../../assets/image";
+import "./product-card.css";
+
+const ProductCard = ({ card }) => {
+   //calculating the discount percent of products
+   const discount = () => {
```

Comment:
Suggestion: Can reduce the number of lines for this function.

Reply:
Thanks for your suggestion